robottelo/constants.py

```diff
@@ -1778,3 +1778,14 @@
     "protocol": "openid-connect",
     "protocolMapper": "oidc-audience-mapper",
 }
+
+RHSSO_NEW_USER = {
+    "email": "test_user@mailinator.com",
```

Comment:
As this is going to be overwritten anyway, could you use `@example.com`?
We don't really care about this value, so I don't see a reason to use any specific service.

Reply:
sure I can